cdk/helpers.py

```diff
@@ -0,0 +1,74 @@
+import boto3
+import jmespath
+from os import getenv as _getenv
+from functools import lru_cache
+from functions.common import zoom_api_request
+
+AWS_PROFILE = _getenv("AWS_PROFILE")
+if AWS_PROFILE is not None:
+    boto3.setup_default_session(profile_name=AWS_PROFILE)
+
+# wrap the default getenv so we can enforce required vars
+def getenv(param_name, required=True):
+    val = _getenv(param_name)
+    if required and (val is None or val.strip() == ''):
+        raise Exception(f"Missing environment variable {param_name}")
+    return val
```

Comment:
this code is repeated in `tasks.py` but if we wanted to consolidate i'm not sure which would be the best home

Reply:
Maybe here since `tasks.py` is already importing `cdk/names.py`